packages/frontend/src/components/common/TagInput/TagInput.tsx

```diff
@@ -313,6 +307,21 @@ export const TagInput = forwardRef<HTMLInputElement, TagInputProps>(
             return false;
         };
 
+        const handleInputBlur = (event: React.FocusEvent<HTMLInputElement>) => {
+            if (addOnBlur && _searchValue !== '') {
+                handleAddTags([_searchValue]);
```

Comment:
We need to handle multiple values on blur eg: 1, 10, 100
Check how it's handled in `handleInputKeydown` 
```
if (splitChars!.includes(event.key) && length > 0) {
                const tags = getSplittedTags({
                    splitChars,
                    allowDuplicates,
                    maxTags,
                    value: _searchValue,
                    currentTags: _value,
                });
```

Reply:
it is not possible to have `1, and something else in the input after comma` because once you do 1, it will be automatically added

Reply:
makes sense 👍

Reply:
but let me test this more